### PR TITLE
Add Decryption from BigInteger Private Key 

### PR DESCRIPTION
**Pull request by Filmaluco**
Also took the opportunity to update the version number and some dependencies

### CODE REVIEW

File: blockstack-sdk/src/main/java/com/colendi/ecies/Encryption.java [hunk 1]
```diff
@@ -58,6 +58,10 @@ public EncryptedResult encryptWithPublicKey(byte[] plainText, String pubKey) {
         return decrypt(privateKey, formData.getIv(), formData.getEphemPublicKey(), formData.getCiphertext(), formData.getMac());
```
Comment (friedger) [inline blockstack-sdk/src/main/java/com/colendi/ecies/Encryption.java:58]: You should call the new method here

**Comment by amrro** [comment]
Hello @friedger, I see that you are managing dependencies in each module separately. 
a gradle file can be used to manage all dependencies and versioning in all modules, like `versions.gradle` or something.

I can start in making a PR, is that welcomed?

**Comment by friedger** [comment]
@amrro yes, please!

**Review by friedger — APPROVED** [review]
Looks good

**Pull request merged**